model-optimizer/mo/utils/custom_replacement_config.py

```diff
@@ -394,3 +391,30 @@ def generate_pattern_for_node(graph: Graph, sub_graph_pattern: str, node_name: s
 
     raise RuntimeError('The pattern that uniquely identifies node "{}" using sub-graph pattern "{}" has not been found'.
                        format(node_name, sub_graph_pattern))
+
+
+def load_and_validate_json_config(config_file_name: str):
+    """
+    Reads and validate custom replacement configuration file config_file_name.
+    :param config_file_name: name of the file to read from.
+    :return: The dictionary which was serialize from json config file.
+    """
+    try:
+        with open(config_file_name, 'r') as f:
+            json_config = json.load(f)
+    except Exception as exc:
+        raise Error("Failed to parse custom replacements configuration file '{}': {}. ".format(config_file_name, exc) +
+                    refer_to_faq_msg(70)) from exc
+
+    try:
+        base_dir = get_mo_root_dir()
+        schema_file = os.path.join(base_dir, 'mo', 'utils', 'schema.json')
+        with open(schema_file, 'r') as f:
+            schema = json.load(f)
+            validator = json_validate.compile(schema)
+            data = validator(json_config)
```

Comment:
is `data` the same as `json_config`? Please double check it on OD model

Reply:
I changed behavior of the function and return json_config instead of data. Just because I did not find examples where use return data. But I think it is equivalent.